modules/quadkey/bigquery/lib/quadkey.js

```diff
@@ -117,25 +117,70 @@ export function sibling (quadint, direction) {
     if (direction !== 'left' && direction !== 'right' && direction !== 'up' && direction !== 'down') {
         throw new Error('Wrong direction argument passed to sibling');
     }
-
-    const tile = ZXYFromQuadint(quadint);
-    const z = tile.z;
-    let x = tile.x;
-    let y = tile.y;
-    const tilesPerLevel = 2 << (z - 1);
     if (direction === 'left') {
-        x = x > 0 ? x - 1 : tilesPerLevel - 1;
+        return sibling_left(quadint);
```

Comment:
Maybe we should follow a standard for the function name case. I guess the most common way for us to name the function is camelCase